versions.lock

```diff
@@ -175,9 +175,10 @@ org.apache.zookeeper:zookeeper-jute:3.7.0 (1 constraints: 8a0d3a28)
 org.apiguardian:apiguardian-api:1.1.0 (3 constraints: 5f208479)
 org.aspectj:aspectjrt:1.8.0 (1 constraints: 0b050836)
 org.bitbucket.b_c:jose4j:0.7.7 (1 constraints: 10050736)
-org.bouncycastle:bcmail-jdk15on:1.69 (1 constraints: e4040431)
-org.bouncycastle:bcpkix-jdk15on:1.69 (1 constraints: e4040431)
-org.bouncycastle:bcprov-jdk15on:1.69 (1 constraints: e4040431)
+org.bouncycastle:bcmail-jdk15on:1.70 (1 constraints: e4040431)
+org.bouncycastle:bcpkix-jdk15on:1.70 (2 constraints: 73133c52)
+org.bouncycastle:bcprov-jdk15on:1.70 (3 constraints: 941ff808)
+org.bouncycastle:bcutil-jdk15on:1.70 (1 constraints: 580d0529)
```

Comment:
precommit fails because we are missing a license file for this new bc dependency - all of the BC libraries are MIT although we have them listed as "BSD_LIKE" - you can see the changes I made for it in the other PR.

Reply:
Fixed.